src/lib.rs

```diff
@@ -580,7 +509,7 @@ pub fn termwidth() -> usize {
 /// ```
 ///
 /// [`wrap`]: fn.wrap.html
-pub fn fill<T: WrapOptions>(text: &str, options: T) -> String {
+pub fn fill<'a, S: WordSplitter, T: Into<Options<'a, S>>>(text: &str, options: T) -> String {
```

Comment:
Is the `'a` lifetime related to the elided lifetime for `text`? Could `'a` be turned into `'_` perhaps?

Reply:
Unfortunately not, Rust requires explicit lifetimes here.

Reply:
Okay, no worries!